go.sum

```diff
@@ -262,6 +262,12 @@ github.com/pkg/xattr v0.4.3 h1:5Jx4GCg5ABtqWZH8WLzeI4fOtM1HyX4RBawuCoua1es=
 github.com/pkg/xattr v0.4.3/go.mod h1:sBD3RAqlr8Q+RC3FutZcikpT8nyDrIEEBw2J744gVWs=
 github.com/please-build/gcfg v1.6.0 h1:pwVJohRItsBAkWh7tw3uWUFqk4a53cLl+Wu4VW33tP4=
 github.com/please-build/gcfg v1.6.0/go.mod h1:qwx/xbr5eBETgfBLP9eXDgXQ5VzLQM5dIjcseh3pkVk=
+github.com/please-build/gcfg v1.6.1-0.20220121092333-3b730990cafa h1:etgi05fzq4gn03Q9H0DzWUlR7pmAGlmKTZ+Psa3jswA=
+github.com/please-build/gcfg v1.6.1-0.20220121092333-3b730990cafa/go.mod h1:qwx/xbr5eBETgfBLP9eXDgXQ5VzLQM5dIjcseh3pkVk=
+github.com/please-build/gcfg v1.6.1-0.20220303142845-6feafe84fba0 h1:QF0BqiPP+JB3w/YDt4W9uROfwifLLlXX8VxuZ1jQDmc=
+github.com/please-build/gcfg v1.6.1-0.20220303142845-6feafe84fba0/go.mod h1:qwx/xbr5eBETgfBLP9eXDgXQ5VzLQM5dIjcseh3pkVk=
+github.com/please-build/gcfg v1.6.1-0.20220308170049-10ea9d657afb h1:6HKarWPya5mPR1xHqTqRNAN+Xox2jndkqVKJRJqu2V8=
+github.com/please-build/gcfg v1.6.1-0.20220308170049-10ea9d657afb/go.mod h1:qwx/xbr5eBETgfBLP9eXDgXQ5VzLQM5dIjcseh3pkVk=
```

Comment:
There's quite a big of duplication here. I wonder if `go mod tidy` will get rid of some?